.github/workflows/js_sdk_ci_jsdom.yml

```diff
@@ -0,0 +1,39 @@
+name: Run tests in jsdom environment
```

Comment:
I believe it is possible to utilize `strategy: matrix:` to have all tests in a single file.

Reply:
```yml
jobs:
    build:
        runs-on: ubuntu-latest

        strategy:
            matrix:
                node-version: [14.x, 15.x]
                environment: ["jsdom", "node"]
...

            - run: npm install
            - run: npm run test:${{ matrix.environment }}:all
              env:
                  CI: true
```